react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -19,6 +19,11 @@ export const BOX_BY_LABEL_IDENTIFIER_AND_ALL_PRODUCTS_QUERY = gql`
       }
       numberOfItems
       comment
+      tags {
+        value: id
+        label: name
```

Comment:
Isn't it confusing down the line to have variable names that are different from the original GraphQL field names?

Reply:
@pylipp The alias is used since we just need to display the results as select options